templates/dataset.yaml

```diff
@@ -20,19 +20,55 @@ dataset:
     # where applicable. By the default, it is assumed that your custom dataset uses EFont's variable naming
     # convention. Any other variables in your dataset that cannot be mapped to the EFont variables will be
     # ignored and cannot be used for calculations at this time.
-    variable_name:
```

Comment:
I think we still need a subtree to distinguish configuration parameters for dataset field name mapping. Maybe we do not call it `variable_name` but something like `fields`?